rpcserver.go

```diff
@@ -3112,8 +3163,13 @@ func ValidateAddress(w *wallet.Wallet, chainSvr *chain.Client, icmd interface{})
 
 	switch ma := ainfo.(type) {
 	case waddrmgr.ManagedPubKeyAddress:
+		// Make sure the public key is available, break out if it's not.
+		pubKey, err := ma.ExportPubKey()
+		if err != nil {
+			break
```

Comment:
If we are not changing the signature of ExportPubKey (or PubKey), then we must explicitly check that the error is the one we expect, and return all others.  This is why I had mentioned returning an optional pubkey, but not erroring (because code like this ends up being written and real errors may be thrown away).